### DISCUSSION TITLE
Q-numeric component

### DISCUSSION THREAD

**Discussion by itplayer** [quasarframework/quasar#9248] [General - Components / Directives / etc] [answered]
Hi,
It looks like that in the previous Quasar release, we have a q-numeric component like this 
http://v0-13.quasar-framework.org/components/numeric.html

What's the alternative in the current 1.15? Q-input and type=number?
I think the UI is not the same, any idea?
Thanks.

**Comment by metalsadman** [comment, marked as answer]
@itplayer wrap QInput then use slots to make your own numeric input. ie. https://codepen.io/metalsadman/pen/rNyxwZy?editors=1010

**Reply by itplayer** [reply]
Awesome! Thanks for the info.

**Reply by itplayer** [reply]
@metalsadman 
So then I need to follow this link to register and make this component to my Quasar app, right?
https://quasar.dev/app-extensions/tips-and-tricks/provide-a-ui-component

**Reply by metalsadman** [reply]
yes you can, or just save it as a vue SFC, then use normal import or register it as a global component in a boot file, the way it's done at the bottom of the docs you linked.

**Reply by itplayer** [reply]
@metalsadman Thanks! I've managed to build and use this awesome component. An additional question might be stupid that how to make its height smaller? It looks like the ordinary CSS style height doesn't help.

**Reply by metalsadman** [reply]
> @metalsadman Thanks! I've managed to build and use this awesome component. An additional question might be stupid that how to make its height smaller? It looks like the ordinary CSS style height doesn't help.

use `dense` prop.
